app/views/home/index.html.slim

```diff
@@ -87,3 +88,21 @@
           .load-more__button.opacity-0.overflow-hidden
             == pagy_bootstrap_nav @pagy, pagy_id: "home-page__main__pagy-nav"
 
+        .col-lg-3.col-md-3.mt-2.home-page__top-interacted-tags
+          - @top_interacted_tags.each do |tag|
+            .mb-2.bg-white.border.rounded.shadow-sm
+              .px-4.py-2.border-bottom
+                .fs-5.fw-bold
+                  = link_to tag_path(tag.name), class: "text-reset text-decoration-none" do
+                    = tag.name
+              - tag.posts.accepted.each_with_index do |post, index|
+                - if index >= 5 
+                  - break
```

Comment:
If you want limit 5 post, you can use limit to query directly instead of using index and break 

` tag.posts.accepted.limit(5).each do`